internal/bminventory/inventory.go

```diff
@@ -851,6 +854,13 @@ func (b *bareMetalInventory) GenerateClusterISOInternal(ctx context.Context, par
 		}
 	}
 
+	if params.ImageCreateParams.StaticNetworkConfig != nil {
+		if err := b.staticNetworkConfig.ValidateStaticConfigParams(params.ImageCreateParams.StaticNetworkConfig); err != nil {
+			log.Error(err)
```

Comment:
as i understand error can have list of errors right? maybe we should log them in ValidateStaticConfigParams ?

Reply:
logging them here will produce the same result: it will just print the list of all the error that we found in validation